src/components/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,115 @@
+import { noop, truncate } from 'lodash-es';
+import React, { FunctionComponent } from 'react';
+import { Link } from 'react-router-dom';
+
+import { MetaData, MetaDataItem, TagList, Thumbnail, ToggleButton } from '../..';
+import { formatDate, formatDuration } from '../../helpers/formatting';
```

Comment:
Moet niet intern in deze component gebruikt worden, het is aan de parent om te voorzien dat deze props proper geformat zijn. `<SearchResult>` moet ze enkel renderen zoals hij ze binnenkrijgt.